### DISCUSSION TITLE
Does Bevy Main branch support bevy_rapier2d?

### DISCUSSION THREAD

**Discussion by The-DevBlog** [bevyengine/bevy#2657] [Q&A] [answered]
Curious if bevy main supports bevy_rapier2d. My toml file looks like this and it doesn't seem to want to compile:

```rust
[dependencies]
bevy = {git = "https://github.com/bevyengine/bevy", branch = "main"}
bevy_rapier2d = "0.11.0"
```

**Comment by bjorn3** [comment, marked as answer]
bevy_rapier2d 0.11.0 only works with bevy 0.5, not the main branch bevy. You could either switch to bevy 0.5 or see of bevy_rapier2d has a version compatible with the main branch in it's git repo.

**Reply by The-DevBlog** [reply]
Ah gotcha. Thanks for the heads up!

**Reply by neo97dev** [reply]
It's pretty easy to update it yourself in a fork, I did it here: https://github.com/neo97x/bevy_rapier/commit/afe52ff403853a8da2f371cded2ea540c96abb2e

I've only tested my fork on 3D, not 2D, so you may need to make additional changes. I've also added some of my own things so you may just want to fork `bevy_rapier` yourself and make similar changes as the commit I linked.
